metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java

```diff
@@ -145,6 +145,26 @@ public void run() {
     }
 
     /**
+     * Starts the reporter polling at the given period.
```

Comment:
> Actually, instead of waiting for the reporter to actually do its work, it would be much better to just verify that the method scheduleAtFixedRate() was called with the correct arguments. There is no need to test the executor itself, as that's a collaborator that has already been tested elsewhere.

Agrred. This would work, too.